### ISSUE TITLE
Publish editorial news

### ISSUE THREAD

**Issue by literarymachine** [hbz/oerworldmap#1124]
As an admin of the OER World Map, I want to publish short messages to tease OER World Map content for my users.

**Comment by literarymachine** [comment]
[ ] Add "Message" type and form
[ ] Show new messages to users as single popups
[ ] Show archive of messages under "News" section in main menu

**Comment by formimkontext** [comment]
This is how the News-site could look like, being part of the "static" sites. @karindr can you provide some realistic content (maybe with pictures) as an example of a message?

![20171107-desktop-news](https://user-images.githubusercontent.com/15141949/32516122-8b619208-c402-11e7-8ba6-289e1ec928a3.png)

**Comment by karindr** [comment]
Not before next week, sorry!

**Comment by karindr** [comment]
You can take the teasers and pictures from the blog as example:
https://oerworldmap.wordpress.com/

**Comment by j0hj0h** [comment]
I have in mind, that the news should have a different scope than the blog. My understanding is:

- **NEWS** announce new functionalities / tease new or outstanding content
- **BLOG** teammeetings, conferences, new atlas is out ...

@karindr do you agree? If so, can you maybe select some posts from the blog, that would fit into the **NEWS** – maybe change the title/wording accordingly?

**Comment by formimkontext** [comment]
I agree with @j0hj0h: With the blog we can share meta information about the project OER World Map. News are / should be directly related to the content of the OER World Map. This is what the news section of the new landingpage is about, right?
